content/spaces/index.textile

```diff
@@ -0,0 +1,48 @@
+---
+title: Spaces
+meta_description: "Spaces by Ably enables you to build collaborative environments in your application."
+---
+
+Spaces enables you to build collaborative environments in your application.
+
+Building realtime collaborative applications requires managing two different states; participant state and application state. Both states involve different types of data, and that data is handled differently.
+
+Participant state is the data related to the members collaborating synchronously in an application, such as their online status, their locations and the positions of their cursors. Application state is the data related to the application itself, such as the data in a cell or field, or the description and status of a task.
+
+The requirements in how realtime updates are managed vary between the two different states. Participant state is largely ephemeral and usually doesn't need to be stored or retained. Application state must be sent to your backend servers, validated and then stored in your database.
+
+Ably's Spaces provides high-level APIs for managing the participant state of your application. Each API is optimized based on the payload structure and frequency of updates that are anticipated for that feature. Ably's "Pub/Sub Channels":/products/channels product provides flexible APIs used to manage application state in realtime by enabling communication between your frontend and backend systems, as well as third-party services.
+
+h2(#sdk). Spaces SDK
+
+Spaces is powered by a "separate SDK":/spaces that is built on top of Ably's JavaScript SDK. Use cases include adding live cursors to an interactive whiteboard, building an avatar stack for an application to show who is online, and displaying which UI components users have selected and which ones are locked editing to avoid conflicts.
```

Comment:
```suggestion
Spaces is powered by a "separate SDK":/spaces that is built on top of Ably's JavaScript SDK. Use cases include adding live cursors to an interactive whiteboard, building an avatar stack for an application to show who is online, and displaying which UI components users have selected and which ones are locked for editing to avoid conflicts.
```